### PR TITLE
Modify multiarch-manager-operator to support for 4.17 rc version

### PR DESCRIPTION
**Pull request by deepsm007**
/cc @aleskandro @openshift/test-platform

### CODE REVIEW

File: clusters/build-clusters/build10/multiarch-manager-operator/admin_01-operator.yaml [hunk 1]
```diff
@@ -5,6 +5,30 @@ metadata:
     workload.openshift.io/allowed: management
```
Comment (aleskandro) [inline clusters/build-clusters/build10/multiarch-manager-operator/admin_01-operator.yaml:5]:
```suggestion
    openshift.io/cluster-monitoring: "true"
```

**Review by aleskandro — APPROVED** [review]

**Comment by deepsm007** [comment]
/unhold

Verified on build09, good to merge

**Pull request merged**